src/components/Tiptap/TiptapMenuBar.vue

```diff
@@ -1,108 +1,299 @@
 <template>
   <div class="flex flex-wrap border">
     <tiptap-menu-item
-        :action="() => editor.chain().focus().toggleBold().run()"
```

Comment:
not sure why we have this changes here, this should focus on search not on tip tap